devices/common_patches/f730_f7x2_f7x3_syscfg.yaml

```diff
@@ -0,0 +1,22 @@
+SYSCFG:
+  PMC:
+    _delete:
+      - PB*
+      - I2C*
```

Comment:
I don't understand these deletions, it looks like RM0431 has the I2C and PB fields in PMC, and doesn't have the MII_RMII_SEL? Whereas RM0410 does have MII_RMII_SEL as well as PB and I2C, and only RM0385 (stm32f745, f750, f7x6) has the MII_RMII_SEL but no PB/I2C fields.

Reply:
I don't know why 0385 doesn't have them, but they're not in any version of the RM and no mention of it in the errata... I don't have any of those F7 to test with. Maybe ST's HAL has clues? I think we can leave them out for now and add later if it turns out they should be here.